services/api/src/resources/backup/resolvers.ts

```diff
@@ -151,6 +153,8 @@ export const deleteBackup: ResolverFn = async (
   pubSub.publish(EVENTS.BACKUP.DELETED, R.prop(0, rows));
 
   userActivityLogger.user_action(`User deleted backup '${backupId}'`, {
+    project: '',
```

Comment:
For the majority of our resolvers, we have the capability of knowing what the project is, even if we don't already load that data in each resolver. For `deleteBackup`, we already load the project id in `perms`, so we can also know the name, for example.

I think we loose a lot of the utility of `sendToLagoonLogs` if we don't add the project when we can. If we added project for everything it would be a lot easier (pretty trivial actually?) to show a log in the UI of all related actions.

This def requires a lot more work, so we could tackle in a separate issue?